package.json

```diff
@@ -5,8 +5,8 @@
     "bootstrap": "lerna bootstrap",
     "docs": "./packages/api-docs/bin/doc-component.js './components/**/lib/index.js' './API.md'",
     "docs:components": "lerna run docs --parallel",
-    "eslint": "eslint ./src",
-    "eslint:fix": "eslint --fix ./src",
+    "eslint": "eslint ./packages/**/src/**.js",
```

Comment:
just wondering why this isn't including `components` as well as `packages` like the following line does?

Reply:
that's a mistake!